### PR TITLE
User Circuits page| Favourites page| Project preview modal - New UI

### PR DESCRIPTION
**Pull request by aayush-05**
**Screenshots**
**User Projects Screen**
1.
image
2.
image
**Favourites Screen**
1.
image
2.
image
**Preview Screen**
1.
image

### CODE REVIEW

**Comment by aayush-05** [comment]
I couldn't get any circuit to be saved on the local server to be viewed as a card on this screen. Facing this error, even db:seed did nothing.
image

But, did checked bits of the card design in a separate file so it should work fine. I will commit any fixes if needed once the new UI gets shifted to the deployment branch @tachyons @nitin10s

**Comment by tachyons** [comment]
Which os ?

**Comment by aayush-05** [comment]
> Which os ?

Windows 10

**Comment by aayush-05** [comment]
@tachyons @nitin10s have a look at this. (got screenshots from gitpod)

File: app/views/users/logix/index.html.erb [hunk 1]
```diff
@@ -30,7 +30,14 @@
   <% end %>
 
   <div class ="row">
-    <h2 class="card-title"><%= @user.name %> - Projects | <a href=  "<%= profile_path(@user) %>"> View Profile </a> | <a href=  "<%= user_favourites_path(@user) %>"> View Favourites </a> </h2>
+    <div class="col-xs-8 col-sm-8 col-md-8 col-lg-8">
+      <%= image_tag @user.profile_picture.url(:medium), style: "width: 100px; height: 100px; border: 0px; margin: 10px;" %>
```
[the remark this answers is not in the document]
Comment (aayush-05) [reply app/views/users/logix/index.html.erb]: Made the required changes

**Comment by aayush-05** [comment]
@tachyons hound issues fixed; I will squash the commits once the pr is approved

**Comment by Ananthavijay** [comment]
I think there is no need for the ````border-radius```` and the lines should be in green color.

**Comment by aayush-05** [comment]
I think the modal box looks a little better with curved edges. Also which lines are you referring to @Ananthavijay ?

**Comment by Ananthavijay** [comment]
In the prototype, the modal box doesn't have curved edges. The line which I am saying is this one:
Inked74798568-1b324a80-52f4-11ea-8a2e-f0cf87514f85_LI

**Comment by aayush-05** [comment]
Does these look fine ? @nitin10s 
image
image
(for some reason I couldn't get the buttons in the modal at the center)

**Comment by Ananthavijay** [comment]
Should the buttons ````Launch```` and ````More```` be present in the modal?

**Comment by aayush-05** [comment]
> * The project preview modal should also display the project title along with the buttons. On previewing - basically the project should get enlarged preserving the title and the buttons as the way they are in the compact view. [i.e Remove the text "Project Preview" and add Title and buttons. Keep the × as it is]
@nitin10s asked to do so. I guess they look fine, add a bit of interactivity to the modal.

**Comment by aayush-05** [comment]
@nitin10s pushed the updated changes, I guess now this can be merged.

**Comment by aayush-05** [comment]
@tachyons could you please check why the tests are failing.
image

**Comment by aayush-05** [comment]
@tachyons merge conflicts solved, now this can be merged.

File: app/views/users/logix/index.html.erb [hunk 2]
```diff
@@ -46,43 +53,50 @@
...
+        <div class="embed-responsive embed-responsive-4by3 feature feature-preview">
           <iframe src="" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
         </div>
       </div>
-      <!--<div class="modal-footer">-->
-      <!--<button type="button" class="btn btn-secondary" data-dismiss="modal">close</button>-->
-      <!--</div>-->
+      <div class="modal-footer modal-footer-userprojects">
+        <% if policy(project).user_access? %>
+                  <a href="<%= simulator_edit_path(project) %>" class="btn btn-about btn-userprojects" target="_blank">Launch</a>
```
Comment (tachyons) [inline app/views/users/logix/index.html.erb]: Use 2 space indentation

File: app/views/users/logix/index.html.erb [hunk 3]
```diff
@@ -46,43 +53,50 @@
...
+              <% else %>
+                  <a href="<%= create_fork_project_path(project) %>" class="btn btn-about btn-userprojects" target="_blank">Fork</a>
+              <% end %>
+              <a href="<%= user_project_path(@user,project) %>" class="btn btn-about btn-userprojects" target="_blank">More</a>
+      </div>
     </div>
 
   </div>
 </div>
+<% end %>
```
Comment (tachyons) [inline app/views/users/logix/index.html.erb]: missing indentation

File: app/assets/stylesheets/logixRelease.css [hunk 4]
```diff
@@ -132,6 +132,61 @@ a.btn.btn-outline-danger {
 }
 
 /* New UI */
+.btn-favourite{
+    display: inline;
+    float:right;
```
Comment (tachyons) [inline app/assets/stylesheets/logixRelease.css]: 2 space indentation

File: app/views/users/logix/index.html.erb (hunk 2 above)
Comment (aayush-05) [reply app/views/users/logix/index.html.erb]: Done

File: app/views/users/logix/index.html.erb (hunk 3 above)
Comment (aayush-05) [reply app/views/users/logix/index.html.erb]: Done

File: app/assets/stylesheets/logixRelease.css (hunk 4 above)
Comment (aayush-05) [reply app/assets/stylesheets/logixRelease.css]: Done

**Comment by aayush-05** [comment]
Done @tachyons

**Pull request merged**